tests/qxmppvcardmanager/tst_qxmppvcardmanager.cpp

```diff
@@ -104,5 +107,66 @@ void tst_QXmppVCardManager::testHandleStanza()
     m_client.removeExtension(manager);
 }
 
+void tst_QXmppVCardManager::testSetClientVCard()
+{
+    SKIP_IF_INTEGRATION_TESTS_DISABLED();
+
+    auto *client = new QXmppClient(this);
```

Comment:
What do you think about using an `unique_ptr` here (to avoid the explicit `delete` later, even at if at the cost of an explicit `get()` a couple of lines below for `QSignalSpy`)? I agree this is not a big deal in this particular case, but IMHO this helps a little to keep the code base maintainable.